javascript-pokemon-card-lookup/server.js

```diff
@@ -0,0 +1,31 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+
+const apiKey = process.env.API_KEY;
+
+// let searchTerm = "venusaur" // temp hardcoded
+
+app.get('/search', async (req, res) => {
```

Comment:
consider making a route at `/` that serves your `index.html` file

or you may be able to get express to serve static files with some built-in function, or some express plugin.